.flake8

```diff
@@ -11,11 +11,19 @@ max-complexity = 10
 exclude =
     # No need to traverse our git directory
     .git,
+    .hg,
+    .mypy_cache,
+    .tox,
+    .venv,
```

Comment:
I have a questions.
is this particular to the way you set things up or is there general tooling that does this?

Reply:
- `.mypy_cache` and `.tox` are standard directories
- `.hg` is standard directory, but this repo is a git one so this souldn't be useful (I kept it because it was already present in the `pyproject.toml`)
- `.venv` is not a standard directory (but was present in the `pyproject.toml`), this is anyway useful given a lot of people keep the project's virtualenv inside the repo (I personally call it `venv` instead of `.venv` but given there is no standard on this naming I guess `.venv` is fine)